packages/repo-tools/src/commands/index.ts

```diff
@@ -40,6 +40,12 @@ function registerSchemaCommand(program: Command) {
       'Generates a Typescript file from an OpenAPI yaml spec. For use with the `@backstage/backend-openapi-utils` ApiRouter type.',
     )
     .action(lazy(() => import('./openapi/generate').then(m => m.bulkCommand)));
+
+  openApiCommand
+    .command('lint [paths...]')
+    .description('Lint OpenAPI schemas.')
+    .option('--strict', 'Fail on linting errors.')
```

Comment:
Shouldn't it always be strict? A caller can always choose to ignore non-zero return codes with like `|| :` or whatnot if they want. Or is there a specific use case for still wanting a successful return despite linting errors being present?

EDIT: Oh looking farther down, it's not a matter of errors, but to fail on NON-errors too right? Maybe needs adjustment of wording here, then

Reply:
Maybe "on any linting warnings" would work here?